infra/build/build_status/update_build_status.py

```diff
@@ -0,0 +1,295 @@
+# Copyright 2020 Google Inc.
```

Comment:
Same nit

Reply:
is this file a copy of https://github.com/google/oss-fuzz/blob/370fb73473ca6bcf09b9c9c4c674de15a6ec4573/infra/build/functions/update_build_status.py
Why does it need to be copied? Sorry if I don't have the context to know why.

Reply:
+1 just delete the old copy.